### ISSUE TITLE
Rewind/Fast Forward on Android Phone

### ISSUE THREAD

**Issue by michaelmiscanuk** [yuliskov/SmartTubeNext#651]
Hi. I am using this amazing app on my android phone, but I cannot find out, how to move the video position to the start (or even fast forward/rewind somehow). Can you please help me out?
Thanks

**Comment by costinmx** [comment]
Same discussion here #645 
Just have to read a few titles below, cmon...

**Comment by michaelmiscanuk** [comment]
I can find any similar question regarding rewind on Android Phone App. Please show me. Thanks.

**Comment by michaelmiscanuk** [comment]
Yes, you are right, it is the same.. I was searching for rewind / fast forward. Not seek :) 
Thanks

**Comment by michaelmiscanuk** [comment]
I wonder if there is at least a way to reset the video position. Because youtube remembers the position, so I want to put it on start. Is there a way? Thanks.

**Comment by linuxstv** [comment]
Seems it's solved. I can use the seek bar on my Android now.
